docs/insights/architecture.md

```diff
@@ -136,7 +135,8 @@ This sniff reports use of superfluous prefix or suffix "Exception" for exception
 
 This sniff checks the size of functions
 
-**Insight Class**: `ObjectCalisthenics\Sniffs\Files\FunctionLengthSniff`
+**Insight Class v1.0**: `ObjectCalisthenics\Sniffs\Files\FunctionLengthSniff`
+**Insight Class v2.0**: `SlevomatCodingStandard\Sniffs\Files\FunctionLengthSniff`
 
 <details>
     <summary>Configuration</summary>
```

Comment:
I wonder what we should use in the description below.. The v2 class or the v1 ? WDYT ?

Reply:
@Jibbarth We can use both! It's not consistent with the rest of the doc, but it's clear. Or do you have another idea?

Reply:
No it's good :+1: